go.mod

```diff
@@ -18,5 +18,6 @@ require (
 
 require (
 	github.com/golang/protobuf v1.4.2 // indirect
+	golang.org/x/text v0.13.0 // indirect
```

Comment:
What pulled this in?

Reply:
the new nats.go client `jetstream` module used only in test.